### ISSUE TITLE
[Accessibility][High Contrast] The text of toolStripStatuslabel item is not visible in HC Mode when setting the RenderMode as System & Enable the IsLink

### ISSUE THREAD

**Issue by Zheng-Li01** [dotnet/winforms#14098]
### .NET version

latest .NET 10.0

### Did it work in .NET Framework?

No

### Did it work in any of the earlier releases of .NET Core or .NET 5+?

The issue can reproduce 8.0/9.0 & .NET Framework

### Issue description

The text of toolStripStatuslabel item is not visible in HS Mode when setting the RenderMode as System & Enable the IsLink

https://github.com/user-attachments/assets/a7a2ec47-0d30-458f-a261-904ed2e0e80e

### Steps to reproduce

1. Create a .NET winforms project
2. Add a ToolStrip control, and set the **RenderMode as System**
3. Add a toolStripStatusLabel item for the toolstrip1, and **Enable IsLink** option
4.  Add a app.manifest file for this project, and then uncomment the below code section
```
<!-- Windows 10 -->
<supportedOS Id="{8e0f7a12-bfb3-4fe8-b9a5-48fd50a15a9a}" />
```
5. Build & Run the application
6. Observe the text of toolStripStatuslabel item after mouse on over it 

Or open & build & run the attached reproduce application
[ReproApplication.zip](https://github.com/user-attachments/files/23877599/ReproApplication.zip)

**More info:**
1. The issue only can reproduce RenderMode as System, not reproduce on ManagerRenderMode & Professional modes
2. The issue also can reproduce .NET Framework application
3.  The issue also can reproduce on toolStripStatusLabel1 item for StatusStrip control
